resources/resources.go

```diff
@@ -624,28 +624,29 @@ type Message struct {
 	ArrivedAt          *RFC3339DateTime `json:",omitempty"`
 	AttachmentCount    int              `json:",omitempty"`
 	AttemptCount       int              `json:",omitempty"`
-	CampaignID         int64            `json:",omitempty"`
 	CampaignALT        string           `json:",omitempty"`
-	ContactID          int64            `json:",omitempty"`
+	CampaignID         int64            `json:",omitempty"`
 	ContactALT         string           `json:",omitempty"`
+	ContactID          int64            `json:",omitempty"`
+	CustomID           string           `json:",omitempty"`
 	Delay              float64          `json:",omitempty"`
-	Destination        Destination
-	FilterTime         int     `json:",omitempty"`
-	FromID             int64   `json:",omitempty"`
-	FromALT            string  `json:",omitempty"`
```

Comment:
These fields are valid - https://dev.mailjet.com/email/reference/messages/#v3_get_message_message_ID

Reply:
nvm, they are for `https://api.mailjet.com/v3/REST/messageinformation`